src/libraries/Common/src/Interop/Windows/Advapi32/Interop.NotifyServiceStatusChange.cs

```diff
@@ -0,0 +1,14 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.Win32.SafeHandles;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class Advapi32
+    {
+        [DllImport(Libraries.Advapi32, EntryPoint = "NotifyServiceStatusChangeW")]
+        internal static extern int NotifyServiceStatusChange(SafeServiceHandle serviceHandle, int notifyMask, in SERVICE_NOTIFY notifyBuffer);
```

Comment:
`in` seems wrong because Windows saves the address of the structure and will update its fields before calling the callback.